lighthouse-core/gather/gatherers/dobetterweb/optimized-images.js

```diff
@@ -73,7 +73,9 @@ class OptimizedImages extends Gatherer {
       }
 
       seenUrls.add(record._url);
-      const isOptimizableImage = /image\/(png|bmp|jpeg)/.test(record._mimeType);
+      const isOptimizableImage = record._resourceType &&
```

Comment:
I was thinking about adding a comment here that we should remove it later on but TBH this makes it more robus anyway so LGTM :)